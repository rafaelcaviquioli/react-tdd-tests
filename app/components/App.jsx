import React from 'react';
import Paginacao from './paginacao/paginacao.jsx';

export default class App extends React.Component {
  render() {
    return (
        <Paginacao totalRegistros={10} quantidadePorPagina={3} paginaAtual={2} />
    );
  }
}