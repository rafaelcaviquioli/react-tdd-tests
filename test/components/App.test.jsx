import React from 'react';
import renderer from 'react-test-renderer';
import App from '../../app/components/App';
import jest from 'enzyme';

describe('App', () => {
    it('Teste Hello World', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
    it('adds 2 + 2 to equal 4', () => {
        expect(2 + 2).toBe(4);
    });
});