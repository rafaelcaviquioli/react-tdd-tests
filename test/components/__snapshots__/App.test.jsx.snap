// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Teste Hello World 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    Hello World
  </h1>
</div>
`;
